### ISSUE TITLE
Embedded mongo doesn't work on Linux aarch64

### ISSUE THREAD

**Issue by mhalbritter** [spring-projects/spring-boot#30082]
When running the Spring Boot build with `./gradlew build`, the `EmbeddedMongoAutoConfigurationTests` fail all the time. This is because Embedded Mongo has no support for Linux aarch64 (See https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/379).

We should workaround this by disabling the tests if running on Linux aarch64.

**Comment by mhalbritter** [comment]
`org.springframework.boot.web.embedded.jetty.SslServerCustomizerTests#whenHttp2IsEnabledServerConnectorsHasSslAlpnH2AndHttpConnectionFactories` and `org.springframework.boot.web.embedded.jetty.SslServerCustomizerTests#alpnConnectionFactoryHasNullDefaultProtocolToAllowNegotiationToHttp11` also don't work on Linux aarch64, see https://github.com/google/conscrypt/issues/1051

**Comment by sbrannen** [comment]
Yes, I think it's worth [opening an issue](https://github.com/junit-team/junit5/issues/new?assignees=&labels=&template=feature_request.md) for the JUnit team to discuss.

Related issue, though focusing on distributions instead of architectures:

- https://github.com/junit-team/junit5/issues/2713

I think explicit support for architectures stands a better chance than explicit support for distributions, since I see the former as generally more useful/applicable.

**Comment by wilkinsona** [comment]
Thanks, Sam!
